storage/cloud-client/public_access_prevention_test.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import storage_get_public_access_prevention
+import storage_set_public_access_prevention_enforced
+import storage_set_public_access_prevention_unspecified
+
+
+def test_get_public_access_prevention(bucket, capsys):
+    short_name = storage_get_public_access_prevention
```

Comment:
I would prefer to either use the long name and let the autoformatter deal with it, or if that's really unacceptably long, to do the original import differently ("import storage_get_public_access_prevention as storage_get_pap" or something). Not a critical change since it's only in the test - up to you.

Reply:
@andrewsg , I wrote these tests to match the [UBLA tests](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/f0a5cbec18011f956c8d5b795837f1ad7b9d7fbf/storage/cloud-client/uniform_bucket_level_access_test.py#L58) since they're similar features. Those are written with `short_name`. I don't mind changing it, but changing it will create an inconsistency. Is that okay?

Reply:
Sorry I didn't respond here earlier. Sure, fine to keep it for consistency. Thanks!